### DISCUSSION TITLE
Confusing UI elements that affect readability

### DISCUSSION THREAD

**Discussion by zeekhuge** [outline/outline#5891] [General]
As of now, the `Heading` tags create a section, but there is no visual indicator (example a tab-space, etc) to suggest that the content is inside the `Heading`
 
![image](https://github.com/outline/outline/assets/8711218/4d2a791a-6fff-4d87-a656-8e4477ed76f0)

1. Given a long content, when the reader is inside a `Heading` tag, it becomes really confusing to make an intuitive assessment of the current reading position.
2. Adding paragraph headings too, becomes a little confusing, as shown above.

We have been using Outline for quite some time now, and this has been a recurring issue with the users, and this is a pretty important reading issue. Any plans regarding this ?

**Comment by zeekhuge** [comment]
In fact, we have made up this habit of adding `---` (break-line) just after these `H1` headings to have a visual indicator of these sections & improve readability.

**Reply by Limezy** [reply]
We do the exact same !

**Comment by zeekhuge** [comment]
Furthermore, consider the first image vs the second
![image](https://github.com/outline/outline/assets/8711218/79f6ad36-72da-46cc-a6ff-2da5c7d0471b)

![image](https://github.com/outline/outline/assets/8711218/38ca1dc7-941e-469e-8b00-ca2b3849c829)

The 2nd is more readable because the point-list is clearly visible. However, a numbered list offers better surface to be able to refer to certain points.

**Comment by EduNetEurope** [comment]
More space before/after headings would increase the readability: https://github.com/outline/outline/discussions/6384
